wxt_cidrbot/cmd_list.py

```diff
@@ -341,6 +355,14 @@ def message_similarity(self, text_split, word_list, msg_threshold):
                         likely_words.append(key_word)
         return likely_words
 
+    def verify_batch_repos(self, repos):
```

Comment:
why is this function needed?

Reply:
This was something I suggested so it made the code more concise by extracting this functionality. It basically just applies regex to the repos.